spark/src/main/scala/org/apache/spark/sql/delta/commands/CreateDeltaTableCommand.scala

```diff
@@ -111,6 +112,13 @@ case class CreateDeltaTableCommand(
     } else if (mode == SaveMode.ErrorIfExists && tableExistsInCatalog) {
       throw DeltaErrors.tableAlreadyExists(table)
     }
+    val tableFeatures = TableFeatureProtocolUtils.
+      getSupportedFeaturesFromTableConfigs(table.properties)
+    if (!Utils.isTesting && (tableFeatures.contains(CatalogOwnedTableFeature) ||
+      sparkSession.conf.contains(
+        TableFeatureProtocolUtils.defaultPropertyKey(CatalogOwnedTableFeature)))) {
```

Comment:
Could directly use [`CatalogOwnedTableUtils.defaultCatalogOwnedEnabled`
](https://github.com/delta-io/delta/blame/master/spark/src/main/scala/org/apache/spark/sql/delta/coordinatedcommits/CoordinatedCommitsUtils.scala#L243)

```suggestion
      CatalogOwnedTableUtils.defaultCatalogOwnedEnabled(spark = sparkSession))) {
```